### DISCUSSION TITLE
Assistance Needed with Deepmodelling Software for New CP2K Version (2022.1) Output

### DISCUSSION THREAD

**Discussion by anmol-compchem** [deepmodeling/dpdata#499] [Q&A]
Dear Deepmodeling Community,

I hope this email finds you well. I am writing to seek assistance with an issue I am facing while using the Deepmodelling software for training a water trajectory from a CP2K AIMD run. Specifically, I am encountering an error when attempting to convert data using the provided script. I have successfully used this script with the output of an older version of CP2K (7.1), but with the current version (2022.1), it is throwing the following error:
# the data contains 0 frames
Traceback (most recent call last):
  File "/home/anmol/work/deep_md/00.data/readdata.py", line 12, in <module>
    data_training = data.sub_system(index_training)
  File "/home/anmol/.local/lib/python3.10/site-packages/dpdata/system.py", line 503, in sub_system
    new_shape = [slice(None) for _ in self.data[[tt.name](http://tt.name/)].shape]
AttributeError: 'list' object has no attribute 'shape'

It appears that the issue might be related to the data containing zero frames, but I am unsure of how to resolve it. I have attached the relevant files for your reference.

readdata.py: The script I am using for data conversion.
Output file from CP2K version 2022.1(water): This is the file I am trying to process and convert using the script.
Output file from CP2K version 7.1(CH4): I successfully processed this file using the same script for comparison.
If anyone in the community has encountered a similar problem or is familiar with this issue, I would greatly appreciate your insights and guidance. Your expertise would be of immense help in resolving this matter and enabling me to proceed with my research.

Thank you very much for your time and consideration.

Best regards,
Anmol

please find the files in the following drive link https://drive.google.com/file/d/1rny_fx4l4OGbthf6YWqfIAM23K-YmzYC/view?usp=drive_link

**Comment by robinzyb** [comment]
hope this helpful 
https://github.com/robinzyb/cp2kdata/issues/15

**Comment by robinzyb** [comment]
Hello, If this discussion is not longer active until tomorrow, it will be closed.
